movies/models.py

```diff
@@ -26,8 +26,12 @@ def get_imdb_url(self):
         return f'{settings.IMDB_URL}{self.imdb_id}'
 
 
+class Csv(models.Model):
```

Comment:
Storing the imported file doesn't have add much value to the application; we want to be able to add multiple movies easier, but after that, we don't really need the csv file anymore. If someone wants to have a history of the uploads, he can keep them locally or, if the project really asks for them for extra operations, then we can store them.
But we shouldn't store more than we need.